u2flib_server/__init__.py

```diff
@@ -25,4 +25,4 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-__version__ = "3.2.1-dev"
+__version__ = "3.3.0-dev"
```

Comment:
I think that changing the backend crypto library counts as backward incompatible, so following http://semver.org/ principles we should increment to major version to 4.0.0. Particularly since `RawRegistrationResponse` objects expose the underlying certificate instances in the  `.certificate` attribute.